internal/dao/sqlx/reservation_dao.go

```diff
@@ -10,23 +12,27 @@ import (
 )
 
 const (
-	createReservation         = `INSERT INTO reservations (provider, account_id, status) VALUES ($1, $2, $3) RETURNING *`
-	createAwsDetail           = `INSERT INTO aws_reservation_details (reservation_id, pubkey_id, source_id, instance_type, amount, image_id, poweroff, name) VALUES ($1, $2, $3, $4, $5, $6, $7, $8)`
-	updateReservationStatus   = `UPDATE reservations SET status = $2 WHERE id = $1 RETURNING *`
+	createReservation         = `INSERT INTO reservations (provider, account_id, step, steps, status) VALUES ($1, $2, $3, $4, $5) RETURNING *`
+	createAwsDetail           = `INSERT INTO aws_reservation_details (reservation_id, pubkey_id, source_id, image_id, detail) VALUES ($1, $2, $3, $4, $5)`
+	updateReservationStatus   = `UPDATE reservations SET status = $2, step = step + $3 WHERE id = $1 RETURNING *`
 	updateReservationIDForAWS = `UPDATE aws_reservation_details SET aws_reservation_id = $2 WHERE reservation_id = $1 RETURNING *`
-	finishReservationStatus   = `UPDATE reservations SET status = $2, success = $3, finished_at = now() WHERE id = $1 RETURNING *`
+	finishReservationSuccess  = `UPDATE reservations SET success = $2, finished_at = now() WHERE id = $1 RETURNING *`
```

Comment:
similar comment here, should we have this static? Here it's just about convenience, not a big deal, but seems like extra argument.

Reply:
I am not sure I understand here.